test/irb/test_irb.rb

```diff
@@ -70,11 +70,10 @@ def test_empty_input_echoing_behaviour
         type "exit"
       end
 
-      # Input cramped together due to how Reline's Reline::GeneralIO works
-      assert_include(
-        output,
-        "irb(main):001> irb(main):002> irb(main):002>  irb(main):002>   => nil\r\n"
-      )
+      # Ignore ANSI color codes
```

Comment:
In IRB we disable coloring when the terminal is `dumb`, which the integration tests use. I feel we should match that behaviour in Reline as well, or make coloring controllable from IRB.

Reply:
This part is acutaly ignoring color reset code `"\e[0m"`, not color specifying code.
Updated to:
```ruby
# Ignore ANSI color reset escape sequences
output = output.gsub(/\e\[0?m/, '')
```
Do you think Reline should not send color reset code when TERM is dumb?

Reply:
This part can be removed after https://github.com/ruby/reline/pull/661

Reply:
removed